src/v/serde/serde.h

```diff
@@ -75,6 +75,9 @@ struct header {
     checksum_t _checksum;
 };
 
+template<typename T>
+concept always_false = false;
+
```

Comment:
this is defined in the second patch, but used in the first

Reply:
removed this stuff by moving assert to the top.